ui/src/translations/fr.json

```diff
@@ -134,5 +134,6 @@
   "prometheus_not_available":"Prometheus n'est pas disponible.",
   "please_refer_to":"Veuillez vous référer à",
   "import_solution_hint":"Vous devez importer une solution pour préparer votre environnement",
```

Comment:
Those three lines were edited in the 2.4 version, but I guess it's really minor... :)

Reply:
Good catch!!!
Made the tiny change!!!